### PR TITLE
Update UG to include Prefix Summary

### LINKED ISSUE #139: Update UG - Prefix Summary

### PR DESCRIPTION
**Pull request by aexolate**
Closes #139

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -64,6 +64,21 @@ If you are an **experienced user**:
 
 1. Refer to the [Features](#features) below for details of each command.
 
+--------------------------------------------------------------------------------------------------------------------
+
+## Prefix Summary
+| Prefix | Attribute   | Constraint                                                   |
+|--------|-------------|--------------------------------------------------------------|
+| `\d`   | Department  | Alphanumeric characters and whitespaces only                 |
```
Comment (Weiennn) [inline docs/UserGuide.md:72]: should we keep it consistent and use an alphabet followed by the forward slash instead? e.g. "n/" and "i/"

**Review by Weiennn — APPROVED** [review]
other than that LGTM!

File: docs/UserGuide.md [hunk 2]
```diff
@@ -64,6 +64,21 @@ If you are an **experienced user**:
...
+| `\e`   | Email       | Valid email address format                                   |
+| `\i`   | Employee ID | Alphanumeric characters only                                 |
+| `\n`   | Name        | Alphanumeric characters and whitespaces only                 |
+| `\p`   | Phone       | Numerical with at least 3 digits                             |
+| `\r`   | Phone       | Alphanumeric characters and whitespaces only                 |
```
Comment (Goh-Li-Ting) [inline docs/UserGuide.md:77]: There seems to be a typo here, should be `Role` instead of `Phone`

**Review by Goh-Li-Ting — APPROVED** [review]
LGTM

**Pull request merged**